### DISCUSSION TITLE
password confirmation with multi guard

### DISCUSSION THREAD

**Discussion by walide67** [laravel/framework#33291] [General]
- Laravel Version: 6.18.15
- PHP Version: 7.3.11
- mysql  Ver 15.1 Distrib 10.4.11-MariaDB

i want to implement password confirmation in some route in my application with multi gaurd but i can't pass the guard name in this function, so it use the default guard

how i can fix that

**Comment by ryangjchandler** [comment]
@walide67 This is better suited for the [Discussions](https://github.com/laravel/framework/discussions) area.
